core/editor/config-providers/editor-v2-config-provider.php

```diff
@@ -129,26 +106,34 @@ public function get_additional_template_paths() {
 		return Editor_Common_Configs::get_additional_template_paths();
 	}
 
-	private function get_packages_script_assets() {
-		if ( ! $this->packages_script_assets ) {
-			$this->packages_script_assets = Collection::make( [ static::APP_PACKAGE ] )
-				->merge( static::EXTENSION_PACKAGES )
-				->merge( static::UTIL_PACKAGES )
-				->map_with_keys( function ( $package_name ) {
-					$assets_path = ELEMENTOR_ASSETS_PATH;
-					$script_asset_path = "{$assets_path}js/packages/{$package_name}.asset.php";
-
-					if ( ! file_exists( $script_asset_path ) ) {
-						return [];
-					}
-
-					/** @var array{ handle: string, deps: string[] } $script_asset */
-					$script_asset = require $script_asset_path;
-
-					return [ $package_name => $script_asset ];
+	private function get_packages_data() {
+		if ( ! $this->packages_data ) {
+			// Loading the file that is responsible for registering the packages in the filter.
+			require_once ELEMENTOR_ASSETS_PATH . 'js/packages/loader.php';
+
+			$packages_data = apply_filters( 'elementor/packages/config', [] );
+
+			// Packages that there is no access to the package.json file.
+			$type_exceptions = [
+				'@elementor/ui' => static::PACKAGE_TYPE_UTIL,
+			];
```

Comment:
Why not? maybe `util` should be the default type, because `util` is like a normal package without side effects

Reply:
Done.